AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRColorGradingAdded.py

```diff
@@ -146,42 +304,427 @@ def AtomEditorComponents_HDRColorGrading_AddedToEntity():
         # 7. Verify HDR Color Grading component is enabled.
         Report.result(Tests.hdr_color_grading_enabled, hdr_color_grading_component.is_enabled())
 
-        # 8. Enable the "Enable HDR Color Grading" parameter.
+        # 8. Toggle "Enable HDR Color Grading".
         hdr_color_grading_component.set_component_property_value(
             AtomComponentProperties.hdr_color_grading('Enable HDR color grading'), True)
-        Report.result(Tests.enable_hdr_color_grading_parameter_enabled,
+        Report.result(Tests.toggle_enable_parameter_on,
                       hdr_color_grading_component.get_component_property_value(
                           AtomComponentProperties.hdr_color_grading('Enable HDR color grading')) is True)
 
-        # 9. Enter/Exit game mode.
+        # 9. Color Adjustment|Weight (float range 0.0 to 1.0, default 1.0)
+        # Set Color Adjustment|Weight to its minimum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Color Adjustment Weight'), 0.0)
+        Report.result(Tests.color_adjustment_weight_min_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Color Adjustment Weight')) == 0.0)
```

Comment:
with floats you may potentially want to use azlmbr.math.Math_IsClose to compare, but if this reliably passes it's probably fine to do a basic equality test like this.

Reply:
I've never seen these float values have issue when setting to absolute values (minimum and maximum). If we were checking somewhere in the middle like 0.123456 we probably should use IsClose, but that doesn't really qualify as a P1 test.